src/App.js

```diff
@@ -12,7 +14,8 @@ function App() {
         <NavLink />
         <Routes>
           <Route path="/rockets" element={<Rockets />} />
-
+          <Route path="/Missions" element={<Missions />} />
```

Comment:
- [ ] As part of the requirements you should add unit & integration tests for  for all your components and functions. kindly add that

![image](https://github.com/Javed-Shahbaz/react-group-project/assets/89282221/82ee6766-8f18-4846-b7ac-a60d9b6e31ab)